templates/accounts/describe_sounds.html

```diff
@@ -297,6 +307,16 @@ <h3>Geotag</h3>
                                 $("#{{form.geotag.zoom.auto_id}}").attr('disabled', false);
                             }
                         });
+                        $("#{{form.geotag.lon.auto_id}}").change(function (event) {
+                            centerMapToLatLonZoomFields({{forloop.counter}});
```

Comment:
Does this mean that the map is going to move every time that you press a key? This might be a bit annoying. I'm not sure what the better solution is. What's the result if it's deleted completely?
We could use blur(), when focus is removed from the element, but that's not great either because you have to click out for it to trigger. What about a timeout, to allow people to delete and write, but not have it update immediately? 
Do we expect people to paste values into here? A paste event might be useful too?

Reply:
The event is not triggered until the element uses focus so the map is not constantly changing. This is as described in [JQuery docs](https://api.jquery.com/change/) (and as I have experienced as well) and is also the normal behaviour of the HTML [`onchange` event](https://www.w3schools.com/jsref/event_onchange.asp). 

We might want to use a paste event for the pasting use case, but I don't think this is really necessary as when user clicks somewhere else the map will be updated anyway.

Reply:
After extra conversations and realizing that only updating the map when input loses focus could result in poor user experience, I updated the geotag edit map in sound edit and sound describe so that map re-sync gets triggered with the `input` event (instead of `change` event). I also added a timeout to perform the actual map syncing to avoid too many events triggering map re-syncs. This also works well for the use case in which a user pastes latitude/longitude or zoom values as `input` event is also triggered.